rubocop-github.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir["README.md", "STYLEGUIDE.md", "LICENSE", "config/*.yml", "lib/**/*.rb", "guides/*.md"]
 
-  s.add_dependency "rubocop", "<=0.82.0"
+  s.add_dependency "rubocop"
```

Comment:
How would you feel about removing the constraints on "rubocop-performance" and "rubocop-rails" as well? My general feeling is that version constraints in gems make upgrades more challenging and don't provide much value except in the case where there is a known conflict between versions.